packages/pg/lib/crypto/sasl.js

```diff
@@ -40,7 +58,34 @@ async function continueSession(session, password, serverData) {
 
   var clientFirstMessageBare = 'n=*,r=' + session.clientNonce
   var serverFirstMessage = 'r=' + sv.nonce + ',s=' + sv.salt + ',i=' + sv.iteration
-  var clientFinalMessageWithoutProof = 'c=biws,r=' + sv.nonce
+
+  // without channel binding:
+  let channelBinding = stream ? 'eSws' : 'biws' // 'y,,' or 'n,,', base64-encoded
+
+  // override if channel binding is in use:
+  if (session.mechanism === 'SCRAM-SHA-256-PLUS') {
+    const peerCert = stream.getPeerCertificate().raw
+    const x509 = await import('@peculiar/x509')
```

Comment:
curious: why make this a lazy/dynamic import?

Reply:
I suppose only that it may often go unused. But that's not very compelling, so I've shifted it to a `require` at the top.